src/radius/wpabuf.h

```diff
@@ -1,153 +1,37 @@
-/**
- * @brief Dynamic data buffer.
- * @file wpabuf.h
- * @author Jouni Malinen
- * @author Alois Klink
- * @copyright
- * SPDX-FileCopyrightText: Copyright (c) 2007-2012, Jouni Malinen <j@w1.fi>
- * SPDX-FileCopyrightText: © 2022 NQMCyber Ltd and edgesec contributors
- * SPDX-License-Identifier: BSD-3-clause
+/*
+ * Dynamic data buffer
+ * Copyright (c) 2007-2012, Jouni Malinen <j@w1.fi>
+ *
+ * This software may be distributed under the terms of the BSD license.
+ * See README for more details.
```

Comment:
Please add upstream source of this file (e.g. git commit and URL)

Reply:
Ok.